.ci/scripts/build.sh

```diff
@@ -160,21 +153,13 @@ fi
 echo "Calling make"
 echo $CXX
 echo $CC
-echo make ${target:-daal_c} ${make_op} \
+echo make ${target:-daal} ${make_op} \
     COMPILER=${compiler} \
-    REQCPU="${optimizations}" \
-    BACKEND_CONFIG="${backend_config}" \
-    PLAT=$PLAT
-make ${target:-daal_c} ${make_op} \
+    REQCPU=\"${optimizations}\" \
+    BACKEND_CONFIG=${backend_config} \
+    PLAT=${PLAT}
+make ${target:-daal} ${make_op} \
     COMPILER=${compiler} \
     REQCPU="${optimizations}" \
-    BACKEND_CONFIG="${backend_config}" \
+    BACKEND_CONFIG=${backend_config} \
     PLAT=${PLAT}
-err=$?
-
-if [ ${err} -ne 0 ]; then
-    status_ex="$(date +'%H:%M:%S') BUILD FAILED with errno ${err}"
-    GLOBAL_RETURN=${err}
-fi
-
-exit ${GLOBAL_RETURN}
```

Comment:
this is no longer necessary due to `set -eo pipefail`?

Reply:
Yeah, that's right. The script will exit at any error (including in pipes) now. That requires a bit of care with things like grep, which return non-zero values on success, but is fine for the scripts as they stand.